Completely remove GTK check.

Works around failing autoreconf:
configure.in:221: error: possibly undefined macro: AM_PATH_GTK

--- a/configure.in
+++ b/configure.in
@@ -208,13 +208,6 @@ AC_ARG_ENABLE(gtk_player,
 [  --enable-gtk-player     build a GTk sample SMPEG player [default=yes]],
               , enable_gtk_player=yes)
 have_gtk=no
-if test x$enable_gtk_player = xyes; then
-    AM_PATH_GTK(1.2.1, have_gtk=yes)
-    if test x$have_gtk = xyes; then
-        CFLAGS="$CFLAGS $GTK_CFLAGS"
-    fi
-    AC_SUBST(GTK_LIBS)
-fi
 AM_CONDITIONAL(HAVE_GTK, test x$have_gtk = xyes)
 
 dnl See if we can build the Mesa player
